src/test/java/seedu/guest/model/guest/UniqueGuestBookTest.java

```diff
@@ -163,4 +164,16 @@ public void asUnmodifiableObservableList_modifyList_throwsUnsupportedOperationEx
         assertThrows(UnsupportedOperationException.class, ()
             -> uniqueGuestList.asUnmodifiableObservableList().remove(0));
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments.